File: src/app/Models/address.ts
export class Address {
   
    country: string; 
    fullName: string; 
    mobileNumber: string; 
    pincode: string;    
    line1: string; 
    line2: string; 
    landmark: string; 
    city: string; 
    state: string; 


    constructor(
        country: string, 
        fullName: string, 
        mobileNumber: string, 
        pincode: string,    
        line1: string, 
        line2: string, 
        landmark: string, 
        city: string, 
        state: string,
    ) {
        this.country = country;
        this.fullName = fullName;
        this.mobileNumber = mobileNumber;
        this.pincode = pincode;
        this.line1 = line1;
        this.line2 = line2; 
        this.landmark = landmark;
        this.city = city;
        this.state = state;
    }   

}

File: GoServer/Model/address.go
package Model


type Address struct {

	Country        		string      `json:"country" bson:"country"`
	FullName        	string      `json:"fullName" bson:"fullName"`
	MobileNumber 		  string 		  `json:"mobileNumber" bson:"mobileNumber"`
	Pincode			 	    string		  `json:"pincode" bson:"pincode"`
	Line1        		  string      `json:"line1" bson:"line1"`
	Line2        		  string      `json:"line2" bson:"line2"`
	Landmark 			    string 		  `json:"landmark" bson:"landmark"`
	City			 	      string		  `json:"city" bson:"city"`
	State        		  string      `json:"state" bson:"state"`

}
